Patch-Source: https://gitlab.freedesktop.org/pipewire/pipewire/-/merge_requests/1664
--
From 193384b26aba3917d086ac3f009aa2cab9d197d2 Mon Sep 17 00:00:00 2001
From: "Tom A. Wagner" <tom.a.wagner@protonmail.com>
Date: Mon, 24 Jul 2023 12:48:33 +0200
Subject: [PATCH] pipewire: add missing stdbool.h include to version.h.in

The header file only works when included in a file where stdbool.h was already included otherwise.
---
 src/pipewire/version.h.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/pipewire/version.h.in b/src/pipewire/version.h.in
index 87fc0cfad..94c645b3a 100644
--- a/src/pipewire/version.h.in
+++ b/src/pipewire/version.h.in
@@ -11,6 +11,8 @@
 extern "C" {
 #endif
 
+#include <stdbool.h>
+
 /** Return the version of the header files. Keep in mind that this is
 a macro and not a function, so it is impossible to get the pointer of
 it. */
-- 
GitLab

